superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -171,11 +171,12 @@ def _inference_step(self, precision):
             self._model.eval()
             while True:
                 for idx, sample in enumerate(self._dataloader):
-                    torch.cuda.synchronize()
                     start = time.time()
-                    sample = sample.cuda()
+                    if self._gpu_available:
+                        sample = sample.cuda()
```

Comment:
add tests with `--no-gpu`?

Reply:
Better to choose one cheap model to test with '--no-gpu', I will use CNN model to test this option.

Reply:
use `--batch-size 0` and `--no-gpu`? so that it will be covered in tests but won't run actually

Reply:
I have add --no-gpu test case in CNN tests, please check.